open-apparel-registry/open-apparel-registry
# Rename "Organization" to "Contributor"

## Overview

Since individuals such as researchers, students, or journalists may contribute to the site, in addition to organizations such as a brand, retailer, manufacturer, etc., we switch to using the more inclusive term "Contributor" throughout the code.

This also brings the back-end terminology in line with what the UI already uses.

organization → contributor
org → contrib

Connects #185 

## Demo

```sql
\d api_contributor;
                                           Table "public.api_contributor"
       Column       |           Type           | Collation | Nullable |                   Default
--------------------+--------------------------+-----------+----------+---------------------------------------------
 id                 | integer                  |           | not null | nextval('api_contributor_id_seq'::regclass)
 name               | character varying(200)   |           | not null |
 description        | text                     |           | not null |
 website            | character varying(200)   |           | not null |
 contrib_type       | character varying(200)   |           | not null |
 other_contrib_type | character varying(200)   |           |          |
 created_at         | timestamp with time zone |           | not null |
 updated_at         | timestamp with time zone |           | not null |
 admin_id           | integer                  |           | not null |
Indexes:
    "api_contributor_pkey" PRIMARY KEY, btree (id)
    "api_contributor_admin_id_key" UNIQUE CONSTRAINT, btree (admin_id)
Foreign-key constraints:
    "api_contributor_admin_id_3bfe88da_fk_api_user_id" FOREIGN KEY (admin_id) REFERENCES api_user(id) DEFERRABLE INITIALLY DEFERRED
Referenced by:
    TABLE "api_facilitylist" CONSTRAINT "api_facilitylist_contributor_id_c09fdaf8_fk_api_contributor_id" FOREIGN KEY (contributor_id) REFERENCES api_contributor(id) DEFERRABLE INITIALLY DEFERRED
```

## Notes

Civil Society Organizations are still called such, and were not switch to Civil Society Contributors.

## Testing Instructions

* Checkout this branch and `manage resetdb`
* Make an account and upload a file. Ensure it works correctly.
* Browse for location on the map. Ensure it works correctly.
